scripts/migrate.sh

```diff
@@ -37,7 +38,7 @@ solana withdraw-from-vote-account test-ledger/vote-account-keypair.json \
 ###############################################################################
 
 # create new v2 accounts
-../solido/target/debug/solido \
+../solido_v2/target/release/solido \
```

Comment:
You should keep a `debug` instead, because this script is for testing and quick checking purposes only. But you can edit user instructions the way you wish